src/templates/zh-TW/2020/methodology.html

```diff
@@ -0,0 +1,476 @@
+{% extends "base/methodology.html" %}
+
+{% block title %}統計方法 | Web Almanac精粹至HTTP Archive{% endblock %}
+
+{% block description %}敘述2020年Web Almanac網路年鑑如何融合：資料集和工具與專案如何運作。{% endblock %}
```

Comment:
個人習慣翻譯方式是按照[中文文案排版指北](https://github.com/sparanoid/chinese-copywriting-guidelines)，裡面有提到「`中英文之間需要增加空格`」及「`中文與數字之間需要增加空格`」，也許這樣排版會更不錯？

```suggestion
 {% block description %}敘述 2020 年 Web Almanac 網路年鑑如何融合：資料集和工具與專案如何運作。{% endblock %}
```

Reply:
很棒的指北，若方便，能否指導下在中英文中增加空白的[工具](https://github.com/hjiang/scripts/blob/master/add-space-between-latin-and-cjk)，感謝。

Reply:
> 很棒的指北，若方便，能否指導分享在中英文中增加空白的自動工具，感謝。

啊 抱歉，沒有實際用過裡面介紹的自動工具 QQ 也許可以試試看 [vinta 寫的 extension](https://github.com/vinta/pangu.js) 🤔?